src/run/handlers/request-context.cts

```diff
@@ -11,7 +11,7 @@ export type RequestContext = {
   responseCacheGetLastModified?: number
   responseCacheKey?: string
   responseCacheTags?: string[]
-  usedFsRead?: boolean
+  usedFsReadForNonFallback?: boolean
```

Comment:
not blocking, but consider calling this something like `usedFsReadForStaticRoute`

Reply:
I think I'll merge this as-is to get this over with. 

Minor side comment is that it hopefully now is actually functionally equivalent ( `usedFsReadForNonFallback` <=> `usedFsReadForStaticRoute`) but there's a chance there is another case here that we will be trying to handle in the future :D In this case I can at least say that `usedFsReadForNonFallback` name is actually correct because it literally check if html is fallback or not, but I am actually not 100% sure that all non-fallbacks are actually "StaticRoutes" heh